requirements_dev.txt

```
@@ -0,0 +1,3 @@
+pre-commit==2.16.0
+pylint==2.12.2
+safety==1.10.3
```

I think this makes more sense, as we want to ensure the latest (major) version is installed upon building the `development` target and is used to test everything.

```suggestion
pre-commit~=2.16
pylint~=2.12
safety~=1.10
```